code/mathematical-algorithms/armstrong_numbers/armstrong.rb

```diff
@@ -0,0 +1,26 @@
+#created by AnkDos
+#You can Simply use '**' to cube ,ex. 3**3=27 ,so cube(remainder) can be written as (reminder**3) 
+def cube(num)
+    return num*num*num
+end
+
+def Is_armstrong(num)
+ quotient=1
+ reminder=0
+ sum=0
+ hold=num
+ while(quotient>0)
+   quotient=num/10
+   reminder=num%10
+   sum=sum+cube(reminder)
+   num=quotient
+ end
+if hold==sum
```

Comment:
return hold==sum also works here.

Reply:
yes , But a rubyist will get the trick.Like using '<<' in place of 'push' and etc etc.
I tried making it for the begginers like me.As you can see there is also a useless cube() method too.
Thanks !